pinot-common/src/main/java/org/apache/pinot/common/assignment/InstancePartitionsUtils.java

```diff
@@ -142,9 +142,10 @@ public static void persistInstancePartitions(HelixPropertyStore<ZNRecord> proper
   /**
    * Removes the instance partitions from Helix property store.
    */
-  public static void removeInstancePartitions(HelixPropertyStore<ZNRecord> propertyStore, String instancePartitionsName) {
+  public static void removeInstancePartitions(HelixPropertyStore<ZNRecord> propertyStore,
```

Comment:
Let's return a boolean instead so that the caller can handle the return properly, e.g. the rest API might want to throw exception when the instance partitions do not exist

Reply:
done